addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -168,6 +170,49 @@ const PopupWidget = publicWidget.Widget.extend({
             return;
         }
         this.$target.find('.modal').modal('show');
+        this.releaseFocus = this._trapFocus();
+    },
+    /**
+     * Traps the focus within the modal.
+     * Returns a function that refocuses the element that was focused before the
+     * modal opened.
+     *
+     * @returns {Function}
+     */
+    _trapFocus() {
+        const tabableEls = getTabableElements(this.el);
+        const previouslyFocusedEl = document.activeElement;
+        if (tabableEls.length) {
+            tabableEls[0].focus();
+        } else {
+            this.el.focus();
+        }
+        // The focus should stay free for no backdrop popups.
+        if (this.el.querySelector(".s_popup_no_backdrop")) {
+            return () => previouslyFocusedEl.focus();
```

Comment:
if previouslyFocusedEl is null (no focused active element), this will throw an error

Reply:
My tests showed that if there's no active element, then activeElement is the body. Did you find otherwise?

Reply:
I added a fallback on `document.body`, although from what I could find `null` only occurs on IE 11. (But maybe I don't have the whole picture, so let's play it safe.)